server/webserver.go

```diff
@@ -750,6 +754,32 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 		w.Write([]byte(fmt.Sprintf("\n\nLatestPlaylist: %v", s.LatestPlaylist())))
 	})
 
+	mux.HandleFunc("/logLevel", func(w http.ResponseWriter, r *http.Request) {
+		if vFlag == nil {
```

Comment:
I'm not sure I understand this case. When would there not be a vFlag? And even if there's not, we'd still want to be able to PUT the log level, yeah?

Reply:
> When would there not be a vFlag

It should always be there, just a habit of checking pointer before accessing them.